code/modules/jobs/job_types/roguetown/adventurer/types/antag/knave.dm

```diff
@@ -62,8 +62,8 @@
 			backr = /obj/item/storage/backpack/rogue/satchel
 			beltl = /obj/item/quiver/arrows
 			backpack_contents = list(/obj/item/needle/thorn = 1, /obj/item/natural/cloth = 1, /obj/item/restraints/legcuffs/beartrap = 2) //poacher gets mantraps
-			H.mind.adjust_skillrank_up_to(/datum/skill/combat/bows, 1, TRUE)
-			H.mind.adjust_skillrank_up_to(/datum/skill/combat/swords, 1, TRUE)
+			H.mind.adjust_skillrank_up(/datum/skill/combat/bows, 1, TRUE)
```

Comment:
compile error, guessing cause this is meant to be adjust_skillrank instead of adjust_skillrank_up